### PR TITLE
Test new node using old db

### PR DESCRIPTION
**Pull request by lucafavatella**
Most test code extracted from WIP branch for https://www.pivotaltracker.com/story/show/163423623

### CODE REVIEW

**Comment by lucafavatella** [comment]
In CI, [test job failed](https://circleci.com/gh/aeternity/aeternity/8472):
```
%%% aehttp_integration_SUITE ==> all.channel_websocket_legacy.sc_ws_timeout_open: FAILED
%%% aehttp_integration_SUITE ==> {{connection_died,normal},
 [{aehttp_ws_test_utils,wait_for_msg,5,
                        [{file,"/home/builder/aeternity/apps/aehttp/test/aehttp_ws_test_utils.erl"},
                         {line,296}]},
  {aehttp_ws_test_utils,unregister_test_for_events,3,
                        [{file,"/home/builder/aeternity/apps/aehttp/test/aehttp_ws_test_utils.erl"},
                         {line,225}]},
  {aehttp_integration_SUITE,sc_ws_timeout_open,1,
                            [{file,"/home/builder/aeternity/apps/aehttp/test/aehttp_integration_SUITE.erl"},
                             {line,4914}]},
```

Unrelated to this PR.

**Comment by tolbrino** [comment]
LGTM

**Comment by lucafavatella** [comment]
Additional CI Linux test job [failed](https://ci.appveyor.com/project/aeternity/aeternity/builds/22480880/job/pk0fkakc4pic8173):
```
%%% aecore_pof_SUITE ==> siblings_on_micro_block: FAILED
%%% aecore_pof_SUITE ==> {{badmatch,none},
 [{aecore_pof_SUITE,siblings_common,6,
                    [{file,"/home/appveyor/projects/aeternity/apps/aecore/test/aecore_pof_SUITE.erl"},
                     {line,209}]},
```

File: system_test/aest_db_SUITE.erl [hunk 1]
```diff
@@ -72,3 +96,8 @@ format(Fmt, Args) ->
 node_spec(Name, DbHostPath) ->
     DbGuestPath = "/home/aeternity/node/data/mnesia",
     aest_nodes:spec(Name, [], #{source  => {pull, "aeternity/aeternity:local"}, db_path => {DbHostPath, DbGuestPath}}).
+
+%% Last Roma release.
+roma_node_spec(Name, DbHostPath) ->
+    DbGuestPath = "/home/aeternity/node/data/mnesia",
+    aest_nodes:spec(Name, [], #{source  => {pull, "aeternity/aeternity:v1.4.0"}, db_path => {DbHostPath, DbGuestPath}, config_guest_path => "/home/aeternity/.epoch/epoch/epoch.yaml"}).
```
Comment (zp-sd) [inline system_test/aest_db_SUITE.erl]: `1.4.0` is the last Roma stable release, ok.

**Pull request merged**